rfcs/text/0004-session.md

```diff
@@ -1,79 +1,29 @@
 # 0004: Session
 <!--^ The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC, taking care not to conflict with other RFCs.-->
 
-- Stage: **0 (strawperson)** <!-- Update to reflect target stage -->
-- Date: 7/30/2020 <!-- Update to reflect date of most recent stage advancement -->
+- Stage: **2 (draft)** <!-- Update to reflect target stage -->
+- Date: TBD <!-- Update to reflect date of most recent stage advancement -->
 
-<!--
-Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
--->
+This RFC calls for the addition of session fields to describe events related to
+various types of "sessions" reported by appliances, security devices, systems,
+management portals, applications, etc.
+
+In addition to these fields, a new `event.category` value of "session" should be added.
+Any event that captures information about a session should include "session" in
+the array field `event.category`.
 
 ## Fields
-This RFC calls for the addition of session fields to describe events related to various types of "sessions" reported by appliances, security devices, systems, management portals, applications, etc.
 
 | Field | Description |
 | ----- | ----------- |
-|session.kind:            | local, remote, network
-|session.authorization:   | user, admin, service
-|session.type:            | system, virtual, application, wired, wireless, vpn
+|session.kind:            | system, application, network
+|session.type:            | [ local, remote, virtual, wired, wireless, vpn ]
```

Comment:
IMHO `kind` vs `type` is really hard to reason about / remember. `type` would be more clear if it was named `connection_type`.

Reply:
I like the idea, tho my brain says "local isn't a connection!" but maybe thats ok.

Reply:
I think I've updated these a bit, was hoping to get some feedback before submitting updates.  I'll try and get the revisions up in the near future

Reply:
Sure, I would say that local is a connection, connecting to local host is a connection, just within a single kernel.